### PR TITLE
issue: 769842 Add config file for travis-ci.org

### PR DESCRIPTION
**Pull request by igor-ivanov**
This PR adds special configuration file used by travis-ci.org environment

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/20/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/21/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/22/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/23/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/24/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/25/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/26/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/27/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/28/ for details (Mellanox internal link).

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,181 @@
+sudo: required
```
Comment (OphirMunk) [inline .travis.yml]: Please set commit message in the following format:
issue: <#RM> - <....textual title....>
<blank line>
<Optional: more text describing the commit>

If RM issue does not already exist - please open a new one

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/29/ for details (Mellanox internal link).

**Comment by igor-ivanov** [comment]
@OphirMunk could you stop travis builds at https://travis-ci.org/Mellanox/libvma/pull_requests (only github project admin can do it)

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+          packages:
+          - g++-4.9
+          - libstdc++6-4.9-dbg
+          - libnl-3-200
+          - libnl-3-dev
+          - libnl-route-3-200
+          - libnl-route-3-dev
+          - libibverbs-dev
+          - librdmacm-dev
+    - compiler: gcc
```
Comment (OphirMunk) [inline .travis.yml]: Can you add env gcc-6?
Gcc-6 is used in fedora 24 where libvma is an official package

**Comment by igor-ivanov** [comment]
how to stop travis builds: https://groups.google.com/forum/#!topic/travis-ci/QqPLH_nUHKk

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/32/ for details (Mellanox internal link).

**Comment by igor-ivanov** [comment]
There is an agreement that the project will not use travis

**Pull request closed without merging**